NYPL-discovery/discovery-front-end
# SCC-3988 - Render a basic DSLink when REVERSE_PROXY_ENABLED is true.

**What's this do?**
This PR conditionally renders a basic link (instead of a React link component) for the Home page and Advanced Search links when an environment variable REVERSE_PROXY_ENABLED is set to true. This allows the reverse proxy to be hit which leads the user to the new version of those pages.

I have not been able to get the environment variable to propagate to the app on QA so I added a fallback of true which will allow us to test while I trace the issues with environment variables.

**Why are we doing this? (w/ JIRA link if applicable)**
[SCC-3988](https://jira.nypl.org/browse/SCC-3988)

**Do these changes have automated tests?**
No

**How should this be QAed?**
Clicking the Search and Advanced Search links on https://test-www.nypl.org/research/research-catalog should lead to the new app.

**Dependencies for merging? Releasing to production?**
The reverse proxy should be enabled on production before we release this.
